.env.production.example

```
@@ -0,0 +1,77 @@
+# =============================================================================
+
+# ── Basic Settings ─────────────────────────────────────────────────────────
+ENVIRONMENT=production
+DEBUG=false
+PROJECT_NAME=Ghost Backend
+VERSION=1.0.0
+
+# ── Database (PostgreSQL) ──────────────────────────────────────────────────
+# Prefer DATABASE_URL in production — it overrides all individual DB_* vars.
+# The app converts postgresql:// to postgresql+psycopg:// internally (psycopg v3).
+DATABASE_URL=postgresql://USER:CHANGE_ME@HOST:5432/ghost  # pragma: allowlist secret
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)